posthog/api/test/test_team.py

```diff
@@ -1604,6 +1608,66 @@ def _patch_linked_flag_config(
             assert response.status_code == expected_status, response.json()
             return response
 
+        def test_filters_null_values_from_app_urls(self):
+            response = self.client.patch(
+                "/api/environments/@current/", {"app_urls": ["https://example.com", None, "https://test.com", None]}
+            )
+
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            response_data = response.json()
+            self.assertEqual(response_data["app_urls"], ["https://example.com", "https://test.com"])
+
+            self.team.refresh_from_db()
+            self.assertEqual(self.team.app_urls, ["https://example.com", "https://test.com"])
+
+        def test_handles_all_null_app_urls(self):
+            response = self.client.patch("/api/environments/@current/", {"app_urls": [None, None, None]})
+
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
+            response_data = response.json()
+            self.assertEqual(response_data["app_urls"], [])
+
+            self.team.refresh_from_db()
+            self.assertEqual(self.team.app_urls, [])
+
+        def test_handles_mixed_valid_and_null_values(self):
```

Comment:
ah, it's multiple tests 🙈 
same all the way down it'd be cool if these were parameterised tests

plus... `self.assertFoo` gives worse output than `assert x == foo`